example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
+
+interface UseGravityArgs {
+  moverMass: number;
+  attractorMass: number;
+  attractor: Vector<number>;
+}
+
+const applyForceForStep = (
+  config: UseGravityArgs,
+  mover: Entity,
+  entity: Entity
+) => {
+  const force = forceV({
+    ...config,
+    mover: mover.position
+  });
+
+  return applyForce({
+    force,
+    moverMass: config.moverMass,
+    acceleration: mover.acceleration,
+    velocity: mover.velocity,
+    position: mover.position
+  });
+};
+
+const run = (config: UseGravityArgs) => {
+  const { start, stop } = rAF();
+
+  start((timestamp, lastFrame, mover) => {
+    const diffTime = timestamp > lastFrame + 64 ? 0 : lastFrame;
+    const steps = Math.floor(timestamp - diffTime);
+
+    let entity: Entity;
+
+    if (steps > 0) {
+      for (let i = 0; i < steps; i++) {
+        entity = applyForceForStep(config, mover, entity);
+      }
+    } else {
+      entity = applyForceForStep(config, mover, entity);
+    }
+
+    return entity;
+  });
+
+  setTimeout(() => {
+    stop();
+  }, 1000);
+};
+
+run({
+  moverMass: 20,
+  attractorMass: 10000,
+  attractor: [500, 500]
+});
```

Comment:
do you want some listener to either be added to the config, or be a separate param that observes an update to the `entity` changes?

Reply:
Hmmm, this is a good callout. I'm not totally sure yet. I think I like the idea of passing in the configuration for deriving the gravitational force alone and keeping the `listener` separate from the configuration. I do think eventually we'll want to get to a place where the user does have some access to the `position` of the entity on a frame by frame basis so they can apply styling (or this'll be somethinng the hook does for us). Like we discussed on Slack, having a little example working will help make this a bit clearer.